tests/utils/dockercli/volumelifecycle.go

```diff
@@ -25,16 +25,15 @@ import (
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
-// CreateDefaultVolume is going to create vsphere docker volume with
-// defaults.
-func CreateDefaultVolume(ip, name string) ([]byte, error) {
+// CreateVolume is going to create vsphere docker volume with given name.
+func CreateVolume(ip, name string) ([]byte, error) {
```

Comment:
I've thought about this. I don't think we actually need many different functions to do the same thing - create a volume with different parameters. Even though Go language does not support function overloading, but we can very easily simulate that using a variadic function - a function that accepts a variable number of parameters. This way, we can call the same API to create different volumes based on needs.

So for now I just rename this, and we can change this to a variadic function later whenever we need it.